amazon-cloudwatch-container-insights/k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/prometheus-eks.yaml

```diff
@@ -13,6 +13,9 @@ data:
   # cwagent json config
   cwagentconfig.json: |
     {
+      "agent": {
+        "debug": true
+      },
```

Comment:
Setting `"debug" : true` in a template... Does not seem necessary.
In my opinion "debug" should not be a default.

Reply:
Agreed. Also, the deployment YAMLs that need to be updated are in the Container Insights repo, not this one.